activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/main/java/org/activiti/cloud/services/modeling/rest/api/ProjectRestApi.java

```diff
@@ -163,6 +165,23 @@ void exportProject(
                     required = false,
                     defaultValue = "true") boolean attachment) throws IOException;
 
+    @ApiOperation(
+            tags = PROJECTS,
+            value = "Save an project as zip file with chosen name",
+            notes = "This will create and download the zip with chosen name " +
+                    "containing the project folder and all related models.<br>")
+    @GetMapping(path = "/projects/{projectId}/saveAs")
+    void saveProjectAs(
+            HttpServletResponse response,
+            @ApiParam(value = EXPORT_PROJECT_ID_PARAM_DESCR, required = true)
+            @PathVariable String projectId,
+            @ApiParam(value = PROJECT_NAME_SAVEAS_DESCR)
+            @RequestParam(name = PROJECT_NAME_PARAM_NAME) String name,
+            @ApiParam(ATTACHMENT_API_PARAM_DESCR)
+            @RequestParam(name = EXPORT_AS_ATTACHMENT_PARAM_NAME,
```

Comment:
same, not, export project call, consider renaming constant or creating a new

Again, it's a bit strange to have it as attachment, but that overlaps with my previous comment on the http verb for this method